### PR TITLE
More phpstan fixes

### PR DESCRIPTION
**Pull request by williamdes**
I fixed one actual unreachable code in 59d638d547a72889ad7ff6ae72bda1446ef10175

The other changes are purely phpdocs

That said I am not sure about the phpdoc ones for DatabaseConnector since the comment was `@param Connector $connector DatabaseConnector`. Let me know

### CODE REVIEW

**Comment by williamdes** [comment]
That all for this PR: `Found 423 errors` left 😄 
Mostly phpdocs

File: classes/Database/Mariadb/DomainMapper.php [hunk 1]
```diff
@@ -43,12 +43,13 @@
...
 class DomainMapper implements DomainMapperInterface
 {
+    /** @var \Liuch\DmarcSrg\Database\DatabaseConnector */
     private $connector = null;
 
     /**
      * The constructor
      *
-     * @param Connector $connector DatabaseConnector
+     * @param \Liuch\DmarcSrg\Database\DatabaseConnector $connector
```
Comment (liuch) [inline classes/Database/Mariadb/DomainMapper.php:52]:
```suggestion
     * @param \Liuch\DmarcSrg\Database\DatabaseConnector $connector DatabaseConnector instance of the current database
```

File: classes/Database/Mariadb/ReportLogMapper.php [hunk 2]
```diff
@@ -41,12 +41,13 @@
...
 class ReportLogMapper implements ReportLogMapperInterface
 {
+    /** @var \Liuch\DmarcSrg\Database\DatabaseConnector */
     private $connector = null;
 
     /**
      * The constructor
      *
-     * @param Connector $connector DatabaseConnector
+     * @param \Liuch\DmarcSrg\Database\DatabaseConnector $connector
```
Comment (liuch) [inline classes/Database/Mariadb/ReportLogMapper.php:50]:
```suggestion
     * @param \Liuch\DmarcSrg\Database\DatabaseConnector $connector DatabaseConnector instance of the current database
```

File: classes/Database/Mariadb/ReportMapper.php [hunk 3]
```diff
@@ -46,14 +46,15 @@
...
+    /** @var \Liuch\DmarcSrg\Database\DatabaseConnector */
     private $connector = null;
 
     private static $allowed_domains = null;
 
     /**
      * The constructor
      *
-     * @param Connector $connector DatabaseConnector
+     * @param \Liuch\DmarcSrg\Database\DatabaseConnector $connector
```
Comment (liuch) [inline classes/Database/Mariadb/ReportMapper.php:57]:
```suggestion
     * @param \Liuch\DmarcSrg\Database\DatabaseConnector $connector DatabaseConnector instance of the current database
```

File: classes/Database/Mariadb/SettingMapper.php [hunk 4]
```diff
@@ -40,12 +40,13 @@
...
 class SettingMapper implements SettingMapperInterface
 {
+    /** @var \Liuch\DmarcSrg\Database\DatabaseConnector */
     private $connector = null;
 
     /**
      * The constructor
      *
-     * @param Connector $connector DatabaseConnector
+     * @param \Liuch\DmarcSrg\Database\DatabaseConnector $connector
```
Comment (liuch) [inline classes/Database/Mariadb/SettingMapper.php:49]:
```suggestion
     * @param \Liuch\DmarcSrg\Database\DatabaseConnector $connector DatabaseConnector instance of the current database
```

File: classes/Database/Mariadb/UpgraderMapper.php [hunk 5]
```diff
@@ -41,12 +41,13 @@
...
 class UpgraderMapper implements UpgraderMapperInterface
 {
+    /** @var Connector */
     private $connector = null;
 
     /**
      * The constructor
      *
-     * @param Connector $connector DatabaseConnector
+     * @param Connector $connector
```
Comment (liuch) [inline classes/Database/Mariadb/UpgraderMapper.php:50]:
```suggestion
     * @param Connector $connector DatabaseConnector instance of the current database
```

**Review by liuch — CHANGES_REQUESTED** [review]

**Comment by williamdes** [comment]
Thank you @liuch for making the changes
I was missing time to get back to this this week

**Comment by liuch** [comment]
It's okay.

**Pull request merged**